test/fixtures/get-orderbook-response.json

```diff
@@ -324,9 +324,9 @@
           "value": "0.5",
           "counterparty": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B"
         },
-        "totalPrice": {
+        "price": {
```

Comment:
Yeah, it's not included:
https://github.com/ripple/rippled/blob/develop/src/ripple/rpc/handlers/AccountOffers.cpp#L119-L124